sdk/tables/Azure.Data.Tables/src/Sas/TableUriBuilder.cs

```diff
@@ -192,18 +192,24 @@ private RequestUriBuilder BuildUri()
             if (!string.IsNullOrWhiteSpace(sas))
             {
                 if (query.Length > 0)
-                { query.Append('&'); }
+                {
+                    query.Append('&');
+                }
                 query.Append(sas);
+                query.Insert(0, "?");
             }
 
             // Use RequestUriBuilder, which has slightly nicer formatting
+            if (query.Length > 0)
```

Comment:
?

Reply:
oops - moved things around and left this by accident.